### DISCUSSION TITLE
BloatyNosy 1.0 PREVIEW

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#103] [Announcements]
Hi everyone, it's been a while since there has been any activity on my open-source projects (just can't find the time), and the list of requests, feedback, and issues has grown long. Especially my last project, **BloatyNosy** has gained considerable popularity and has practically become the standard tweaker/debloater in the Windows 11 scene with millions of downloads. 

I have taken this opportunity to give the latter project a more substantial update. I've started working on it in the last few days and would like to present a preview to you. In addition to refreshing the UI, especially the main user controls, the list of changes under the hood is quite extensive. Try it out and send feedback.

![BloatyNosy_aMhi9pcy60](https://github.com/builtbybel/BloatyNosy/assets/57478606/8f66abd1-d810-486b-b677-57784d2471c1)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/BloatyNosy/releases/tag/1.0'>BloatyNosy 1.0 PREVIEW</a>.</em>

**Comment by Newr0n** [comment]
Thank you for the update and your work on this.

**Comment by ryguysenpai** [comment]
Unable to right click and configure in bloatware menu now

**Comment by TBrasse** [comment]
UI seems to be borked on 1.0.241.0
<img width="300" alt="image" src="https://github.com/builtbybel/BloatyNosy/assets/24989058/13378ab3-4e34-4694-a1b1-a792bb2b57df">
and when you resize the window:
<img width="300" alt="image" src="https://github.com/builtbybel/BloatyNosy/assets/24989058/da962e9b-8f83-4ebc-805e-5f0f303eae1f">

**Reply by Belim** [reply]
Fix is on the way in final
